lib/beaker-pe/install/pe_utils.rb

```diff
@@ -1074,6 +1074,16 @@ def register_feature_flags!(opts)
           Beaker::DSL::InstallUtils::FeatureFlags.new(opts).register_flags!
         end
 
+        # PE 2018.1.0 has mco disabled by default. If we are running hosts with
+        # roles hub or spoke then we intend to test mco. In this case we need
+        # to change a setting in pe.conf to allow mco to be enabled.
+        def get_mco_setting(hosts)
+          if(version_is_less('2018.1', hosts[0]['pe_ver']) && (hosts.any? {|h| h['roles'].include?('hub') || h['roles'].include?('spoke')}))
+            return {:answers => {:pe_install => {:disable_mco => false}}}
```

Comment:
Oh, I missed this.  The parameter should be a single string, not a Hash.

{:answers => { 'pe_install::disable_mco' => false }}

Otherwise you are forcing Beaker::Answers to attempt to squash these exploded parameters.  Which it can't do because we have legitimately structured data to pass into pe.conf (feature_flags hash).